generic/install.sh

```diff
@@ -7,13 +7,13 @@ echo "usage: $0 architecture"
 else
 
 TARGET=$1
-V=2.25
+V=2.34
 
 cd /tmp
-wget -nc http://ftp.gnu.org/gnu/binutils/binutils-$V.tar.gz
-wget -nc http://ftp.gnu.org/gnu/binutils/binutils-$V.tar.gz.sig
+wget -nc https://ftp.gnu.org/gnu/binutils/binutils-$V.tar.gz
```

Comment:
Why do this? If we verify the signature later, this encryption seems redundant

Reply:
Just updating what was there.. feel free to change it.